src/components/notes/editor.jsx

```diff
@@ -180,6 +181,27 @@ const Editor = translate()(
       [loading, doc]
     )
 
+    // Be sure to save everything before leaving
+    // and force sync with the io.cozy.file
+    async function forceSync() {
+      if (doc) {
+        // wait for every event to finish
+        const provider = await collabProvider.provider
```

Comment:
Why are we returning : `Promise.resolve(provider)` ? If this task is async we should use a fetcher to tell so. `await collabProvider.fetchProvider()` is clearer. But I'm curious about the promise.resolve

Reply:
`collabProvider`'s API defined by Atlassian. It expects a Promise at the `provider` key. Our own provider is initialized without needing any async code. We use `Promise.resolve()` to comply with Atlassian's API.

Reply:
OK thanks. Adding a comment around `Promise.resolve(provider)` can be useful I think :)